packages/firestore/test/integration/api/validation.test.ts

```diff
@@ -847,8 +847,8 @@ apiDescribe('Validation:', (persistence: boolean) => {
         const reason =
           `Invalid query. You have a where filter with an ` +
           `inequality (<, <=, >, or >=) on field 'x' and so you must also ` +
-          `use 'x' as your first orderBy(), but your first orderBy() is on ` +
-          `field 'y' instead.`;
+          `use 'x' as your first argument to Query.orderBy(), but your first ` +
+          `orderBy() is on field 'y' instead.`;
```

Comment:
It's somewhat surprising that we replace the first `orderBy` but not the second. In other cases where they're not spelled the same this seems like it would allow the implementation to leak out.

Reply:
JavaScript reg-exp only replace one instance by default. I could use `/g` here but then the message will be harder to parse (or has to be rewritten even more).